sdk/storage/storage-blob/.prettierignore

```diff
@@ -0,0 +1,2 @@
+src/generated/**/*.ts
+package-lock.json
```

Comment:
Is prettier not requiring a newline to be there at end of file?

Reply:
This is not a json, ts or js so it doesn't get formatted

Reply:
Fixed

Reply:
Thanks